spec/system/supervisor_views_volunteers_page_spec.rb

```diff
@@ -51,4 +51,17 @@
     expect(page).not_to have_text("Contact Made In Past 60 Days")
     expect(page).not_to have_text("Last Contact Made")
   end
+
+  describe "filters" do
+    let(:supervisor) { create(:supervisor, :with_volunteers) }
+
+    it "by default only shows volunteers filtered by signed in supervisor" do
```

Comment:
Hi @compwron 👋  I think something is off with Travis Ci
if you look at line 7 above in the same file https://github.com/rubyforgood/casa/blob/4475a7aa1cc34f1102af2a4b619a90cd137a5f30/spec/system/supervisor_views_volunteers_page_spec.rb#L7
a pre-existing filter test is commented out with `xit` (with a comment about Travis CI)

in the travis CI report (https://travis-ci.com/github/rubyforgood/casa/builds/188064832):
![image](https://user-images.githubusercontent.com/25035236/95027114-fc967f00-064a-11eb-899a-58a0d87087fa.png)
^ It shows the related filters test as being skipped 

Should I change this test to use `xit`? If you pull the branch, it should meet the requirements as described in the github issue

Reply:
Travis running tests is the source of truth for whether tests work. I would much rather fix an `xit` test than make more of them.

Reply:
![image](https://user-images.githubusercontent.com/25035236/95028326-4fc0ff80-0654-11eb-84a3-296b23ba8f96.png)

@compwron I understand--could you please help provide guidance on how I can add the new test? I'm directly retrieving the number of volunteers under the supervisor and adding 1 (for the header row) so I'm not sure why the test result is showing "expected 3, got 1". 

I am not quite sure how to fix the pre-existing `xit` test on line 7, it mentions "# Add back when Travis CI correctly handles large screen size"

Thank you for your patience 🙏

Reply:
@cdncat the test is depending on a css class which may have changed. I recommend looking at the css classes on the page and picking the one that applies to your field. If you need to add a new class or id, go for it. :)

Reply:
hiya 👋  
I double checked the table `id` and also I tried fixing the new test but with more inspection it seems rspec could be running the test on the initial state of the `/volunteers` page which is different from the final state of the page (once everything is loaded)